.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,36 @@
+name: Express Rescue gitHub actions
+
+on: pull_request
+
+jobs:
+  scripts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Setting up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
```

Comment:
Here we need a matrix cuz we want to prove that express-rescue works against lots of node versions. Not sure if you are familiar with matrix yet, but It looks like this:


```yaml
jobs:
  scripts:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        node:
        - 7
        - 8
        - 9
        - 10
        - 11
        - 12
        - 13
        - 14
        - 15
        - 16
        - 17
        
      steps:
      - # ...
      
      - name: Setting up node
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node }}
```

The package was released when node was at v7 and it doesn't really use nothing fancy, so I guess for this major version of express-rescue we should keep supporting v7. Thus we need to prove that all those versions are supported by testing against them.

Reply:
Yeah, that makes sense! already added to pr!